website/versioned_docs/version-1.0/api/ref/Channel.md

```diff
@@ -0,0 +1,220 @@
+    // handle the instrument
+});
+
+// later
+contactListener.unsubscribe();
+instrumentListener.unsubscribe();
+```
+
+#### See also
+
+* [`Listener`](Types#listener)
+* [`ContextHandler`](Types#contexthandler)
+* [`broadcast`](#broadcast)
+* [`getCurrentContext`](#getcurrentcontext)
+
+### `broadcast`
+
+```typescript
+public broadcast(context: Context): Promise<void>;
+```
+
+Broadcasts a context on the channel. This function can be used without first joining the channel, allowing applications to broadcast on both App Channels and User Channels that they aren't a member of.
+
+If the broadcast is denied by the channel or the channel is not available, the promise will be rejected with an `Error` with a `message` string from the [`ChannelError`](Errors#channelerror) enumeration.
+
+Channel implementations should ensure that context messages broadcast by an application on a channel should not be delivered back to that same application if they are joined to the channel.
+
+If you are working with complex context types composed of other simpler types (as recommended by the [FDC3 Context Data specification](../../context/spec#assumptions)) then you should broadcast each individual type (starting with the simpler types, followed by the complex type) that you want other apps to be able to respond to. Doing so allows applications to filter the context types they receive by adding listeners for specific context types.
+
+#### Example
+
+```javascript
+const instrument = {
+    type: 'fdc3.instrument',
+    id: {
+        ticker: 'AAPL'
+    }
+};
+
+try {
+    channel.broadcast(instrument);
+} catch (err: ChannelError) {
+    // handler errror
+}
+```
+
+#### See also
+
+* [`ChannelError`](Errors#channelerror)
+* [`getCurrentContext`](#getcurrentcontext)
+* [`addContextListener`](#addcontextlistener)
+
+### `getCurrentContext`
+
+```ts
+public getCurrentContext(contextType?: string): Promise<Context|null>;
+```
+
+When a _context type_ is provided, the most recent context matching the type will be returned, or `null` if no matching context is found.
+
+If no _context type_ is provided, the most recent context that was broadcast on the channel - regardless of type - will be returned.  If no context has been set on the channel, it will return `null`.
+
+It is up to the specific Desktop Agent implementation whether and how recent contexts are stored. For example, an implementation could store context history for a channel in a single array and search through the array for the last context matching a provided type, or context could be maintained as a dictionary keyed by context types. An implementation could also choose not to support context history, in which case this method will return `null` for any context type not matching the type of the most recent context.
+
+If getting the current context fails, the promise will be rejected with an `Error` with a `message` string from the [`ChannelError`](ChannelError) enumeration.
```

Comment:
```suggestion
If getting the current context fails, the promise will be rejected with an `Error` with a `message` string from the [`ChannelError`](Errors#channelerror) enumeration.
```

This link is broken. The above suggestion fixes it, which should help fix your build errors. I have tested this change.

Reply:
this was in the docusaurus report.  fixed